substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVSymbolSubrecord.java

```diff
@@ -359,6 +454,31 @@ public String toString() {
         }
     }
 
+    @SuppressWarnings("unused")
+    public static final class CVSymbolUDTRecord extends CVSymbolSubrecord {
```

Comment:
Do you have the unused subclasses for future improvements or are the leftovers from prev. experiments?

Reply:
They are there for future use.